apps/vc-api/src/vc-api/exchanges/dtos/transaction.dto.ts

```diff
@@ -54,6 +54,6 @@ export class TransactionDto {
   // TODO: make generic so that it can be used in all Dtos
   static toDto(transaction: TransactionEntity): TransactionDto {
     const data = classToPlain(transaction);
-    return plainToClass(TransactionEntity, data);
+    return plainToClass(TransactionDto, data);
```

Comment:
This is out of scope of this PR, but a breaking change is coming in the `class-transformer` v0.4.1 library. It is suggested to use plainToInstance instead of plainToClass (https://github.com/typestack/class-transformer/blob/develop/CHANGELOG.md#041-breaking-change---2021-11-20)

Anyway, we will need to refactor the code soon as it is going to break after next dependency upgrade. Semantically, this version should not introduce any breaking changes...

Reply:
We will need to upgrade `class-transformer` anyway: https://github.com/typestack/class-transformer/blob/develop/CHANGELOG.md#050-breaking-change---2021-11-20

Reply:
Yes, they have already messed with this in v0.4.1 and reverted in v0.5.0, but we can expect this at some point, so it is better to get used to using `plainToInstance` and generally not using functions with `Class` in their names if there is an `Instance` equivalent.